test/integration/auth_int_test.go

```diff
@@ -34,11 +34,10 @@ func (suite *AuthIntegrationTestSuite) TestAuth() {
 	suite.OnlyRunForTags(tagsuite.Auth, tagsuite.Critical)
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
-	user := ts.CreateNewUser()
 	ts.LogoutUser()
-	suite.interactiveLogin(ts, user.Username, user.Password)
+	suite.interactiveLogin(ts, e2e.PersistentUsername, e2e.PersistentPassword)
 	ts.LogoutUser()
-	suite.loginFlags(ts, user.Username)
```

Comment:
I feel like there is value in authenticating a new user, rather than an existing one. Is there a specific reason you wanted to change this?

Reply:
Hmm, that's interesting... what specifically is the value you see in creating a new user for each test invocation? To me this is testing that we can log in and out.

Reply:
I'm not sure. It's one of those things: if it's not broken, don't fix it.